docs/users-guide/sql-snippets.md

```diff
@@ -2,7 +2,7 @@
 
 ![Highlight and save as snippet](./images/sql-snippets/highlight_and_save_as_snippet.gif)
 
-**SQL snippets** are reusable bits of SQL or native queries. Anyone with permissions to the [SQL editor](writing-sql.md) can create and edit snippets, which are then available for all SQL authors.
+**[SQL snippets](https://www.metabase.com/blog/sql-snippets/index.html)** are reusable bits of SQL or native queries. Anyone with permissions to the [SQL editor](writing-sql.md) can create and edit snippets, which are then available for all SQL authors.
```

Comment:
After looking at a bunch of these examples where you've linked to a blog article, I think they seem better when they're contextualized as "if you want to learn more…" I.e., it's unclear when just looking at this hyperlink why I should click on it when it seems I'm already on a page which purports to be the place to go to learn about this subject (the doc page itself). The hyperlink as it is here almost makes me wonder if I'm on the wrong page currently — what does _this_ SQL snippets link have that the page I'm on doesn't?